.pre-commit-config.yaml

```diff
@@ -8,26 +8,36 @@ repos:
       additional_dependencies: ['click==8.0.4']
 
 -   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.971
+    rev: v0.982
     hooks:
-    -   id: mypy
-        entry: venv/bin/mypy
-        exclude: '(src/test/|noxfile\.py)|(src/lumigo_opentelemetry/external/.*)'
-        args: ['--explicit-package-bases',  # Needed for src/ci
-               '--namespace-packages',  # Needed for opentelemetry package
-               '--disallow-any-generics', '--disallow-untyped-defs', '--disallow-incomplete-defs', '--check-untyped-defs', '--no-implicit-optional', '--warn-redundant-casts', '--warn-unused-ignores', '--warn-return-any', '--no-implicit-reexport', '--strict-equality', '--ignore-missing-imports']
```

Comment:
Why did you remove "warn-unused-ignores"?

Reply:
Because of a false positive I could not get rid of otherwise.

Reply:
It means that you can remove the `# type: ignore`.